src/main/java/org/dimdev/rift/network/Message.java

```diff
@@ -13,13 +13,14 @@
 import net.minecraft.network.play.server.SPacketCustomPayload;
 import net.minecraft.server.MinecraftServer;
 import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.registry.IRegistry;
 import net.minecraft.util.registry.RegistryNamespaced;
 import net.minecraft.world.WorldServer;
 
 import java.util.function.Predicate;
 
 public abstract class Message {
-    public static final RegistryNamespaced<ResourceLocation, Class<? extends Message>> REGISTRY = new RegistryNamespaced<>();
```

Comment:
Should the "message" string not be namespaced?

Reply:
Should be better, I wasn't sure if it was best to add a new class or use mixin's to add the method straight to IRegistry